content/300-guides/050-database/100-developing-with-prisma-migrate/165-enable-native-database-functions.mdx

```diff
@@ -5,21 +5,25 @@ metaDecription: How to enable native database functions for projects that use Pr
 
 <TopBlock>
 
-<Admonition type="warning">
+Some [native database functions](/concepts/components/prisma-schema/features-without-psl-equivalent#native-database-functions) <span class="concept"></span> are part of optional extensions. You cannot use the function if the extension is not installed. If your project uses [Prisma Migrate](/concepts/components/prisma-migrate), you must install the extension as part of a migration.
 
-**MongoDB not supported** <br />
-Prisma Migrate does not currently support the [MongoDB connector](/concepts/database-connectors/mongodb).
+<Admonition type="info">
 
-</Admonition>
+This guide **does not apply for MongoDB**.<br />
+[MongoDB](/concepts/database-connectors/mongodb) does not support optional extensions as those do not exist for MongoDB.
 
-Some [native database functions](/concepts/components/prisma-schema/features-without-psl-equivalent#native-database-functions) <span class="concept"></span> are part of optional extensions. You cannot use the function if the extension is not installed. If your project uses [Prisma Migrate](/concepts/components/prisma-migrate), you must install the extension as part of a migration.
+</Admonition>
```

Comment:
Rewrote to make clear that extensions are not a thing for MongoDB as they are for relational databases.